src/fontra/views/editor/scene-model.js

```diff
@@ -103,6 +103,12 @@ export class SceneModel {
     return getSelectedGlyphName(this.selectedGlyph, this.glyphLines);
   }
 
+  isSelectedGlyphLocked() {
+    return this.getSelectedPositionedGlyph()?.varGlyph?.glyph.customData?.[
```

Comment:
```suggestion
    return !!this.getSelectedPositionedGlyph()?.varGlyph?.glyph.customData?.[
```
To ensure the return value is a bool (!!undefined -> false)